torch/_dynamo/variables/nn_module.py

```diff
@@ -295,7 +296,11 @@ def call_method(
                 "Must provide a valid source in order to inline, "
                 "since inlined function may have default args which must be guarded."
             )
-            fn = module.forward.__func__
+            if name == "forward":
+                fn = module.forward.__func__
+            else:
+                assert name == "_call_impl"
+                fn = module._call_impl.__func__
             assert istype(fn, types.FunctionType)
             options["source"] = AttrSource(
                 AttrSource(self.source, "forward"), "__func__"
```

Comment:
should update the hardcode ```forward``` to ```getattr(module, name)```.

Reply:
So in this case I was getting a method is not iterable error later down the line but this worked if I just replaced forward by name